README.md

```diff
@@ -71,6 +71,14 @@ To run the unit tests, run:
 npm run test
 ```
 
+To run the unit tests in the Chrome DevTools inspector, run:
+
+```
+npm run test-debug
+```
+
+Then, open `chrome://inspect` in your Chrome browser and select the "node_modules/.bin/jest" target to open the Chrome DevTools. You can set breakpoints in Chrome DevTools or insert a `debugger;` statement into the code to pause execution at that point.
```

Comment:
@thallada this looks great - thanks for doing this!

A couple (very nitty) style things (just my opinion so feel free to not implement these suggestions):

1. Maybe move this to the end of the `Unit Testing` section and then add a `Debug Tests Via Chrome` (sub) header or something? Might help to highlight how to debug tests.
2. Maybe add a screenshot (something like this perhaps?)
![image](https://user-images.githubusercontent.com/8136030/33859377-a2cf9b10-dea1-11e7-920c-3369c38f6c98.png) just to add a little bit more context?